### DISCUSSION TITLE
Protobuf-spoof patch on old YT-versions

### DISCUSSION THREAD

**Discussion by s7san007** [kitadai31/revanced-patches-android6-7#25] [Q&A]
Is it possible to implement the protobuf-spoof patch by manually editing the smali code (copy-paste) or something else? I need YouTube Vanced 16.02.35 with protobuf-spoof. A week ago 16.02.35 stopped working.
![Screenshot_20230430_173128](https://user-images.githubusercontent.com/130248949/235381443-34b3de5a-e68a-4878-9e68-f63c2834d16f.png)

**Comment by kitadai31** [comment]
Maybe you can.

There is a person who implemented protobuf-spoof on YouTube 16.40.36.
https://github.com/d4n3436/revanced-patches-android5

Therefore, I think that protobuf-spoof can be implemented in 16.02.35 as well.

> Please note that you need both `client-spoof` and `protobuf-spoof` patches to avoid the problem.
>
> `client-spoof` -> Remove "The following content is ..." screen
> `protobuf-spoof` -> Prevent infinite buffering issue

Btw, what's the reason why do you need 16.02.35?
